<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="banner.local" name="com.exiu.tester.carowener.my.Test022_MyOrder" tests="1" failures="0" timestamp="03 七月 2017 06:21:36 GMT" time="59.405" errors="1">
  <testcase name="我的订单" time="59.405" classname="com.exiu.tester.carowener.my.Test022_MyOrder">
    <error type="java.lang.Throwable">
      <![CDATA[java.lang.Throwable
	at message : orderName不正确 expected [七座洗车] but found [维修] in method : .我的订单(Test022_MyOrder:31)
	at com.exiu.tester.carowener.my.Test022_MyOrder.我的订单(Test022_MyOrder.java:31)
]]>
    </error>
  </testcase> <!-- 我的订单 -->
</testsuite> <!-- com.exiu.tester.carowener.my.Test022_MyOrder -->
